### DISCUSSION TITLE
JSPsych table format

### DISCUSSION THREAD

**Discussion by gilirusak** [jspsych/jsPsych#3049] [Q&A]
Hi all,

I am trying to develop a table element that has radio buttons within the table, as shown in the attached screenshot. What is the best way to do this using JSPsych elements?
<img width="900" alt="Screen Shot 2023-05-04 at 2 47 37 PM" src="https://github.com/jspsych/jsPsych/assets/35749644/38612986-14f9-45aa-8c34-14d65785481d">

Thanks!

**Comment by jodeleeuw** [comment]
Hi @gilirusak,

This is a trickier question to answer thoroughly because it will involve a lot of CSS formatting work. The best plugin to use for this is probably the [survey-html-form](https://www.jspsych.org/7.3/plugins/survey-html-form/) plugin, which lets you write your own custom HTML with embedded form elements. Then you can set up a standard `<table>` element and embed the input items in it. If you need more advice please feel free to follow up here.
